source/production/F0.Minesweeper.Components/Abstractions/Enums/MouseButtonType.cs

```diff
@@ -0,0 +1,15 @@
+namespace F0.Minesweeper.Components.Abstractions.Enums
+{
+	/// <summary>
+	/// Contains the mouse button type definition according to the documentation of the MouseEventArgs.Button
+	/// property from the Microsoft.AspNetCore.Components.Web assembly.
+	/// Infos taken from <see href="https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.web.mouseeventargs.button?view=aspnetcore-3.1">Microsoft Docs</see>
+	/// on the 4th of November, 2020.
+	/// </summary>
+	public enum MouseButtonType
```

Comment:
I think I'm kind of lost with this comment. Do you mean to create a unit test which confirms the values defined here?

Reply:
yes - that's what I meant!
I should write my sentences just as I prefer my statements: "explicitly typed" ... and, while at it, also refactor my jokes

Reply:
I tried to do the tests but turns out they would be pretty meaningless. I implemented a test component that just had a button with the `onmousedown` event. With `bunit` I can of course trigger this event but I have to provide a `long` value to indicate which mouse button was clicked. 
So if I do this then I would have a test that would state `1 == 1` which is in my opinion pretty meaningless. 

Would you know of a good way to test this?

Reply:
actually, my thinking was quite naive - as usual ;-)
"just" checking the associated integer with the left/middle/right mouse button is indeed 0, 1 and 2 respectively
my motivation here is that we have those specs in code, rather than "just" a comment